<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MacBook-Pro16" failures="2" tests="6" name="petTest" time="3.613" errors="0" timestamp="2024-08-28T14:46:20 EDT">
  <testcase classname="api.Tests.userTests" name="postTest" time="1.918"/>
  <testcase classname="api.Tests.userTests" name="getTest" time="0.127">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.Tests.userTests.getTest(userTests.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getTest -->
  <testcase classname="api.Tests.userTests" name="putTest" time="0.108"/>
  <testcase classname="api.Tests.userTests2" name="postTest" time="0.11"/>
  <testcase classname="api.Tests.userTests2" name="getTest" time="0.091">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.Tests.userTests2.getTest(userTests2.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getTest -->
  <testcase classname="api.Tests.userTests2" name="putTest" time="0.096"/>
</testsuite> <!-- petTest -->
